src/Microsoft.Azure.WebJobs.Host/Config/FluentBindingRule.cs

```diff
@@ -120,11 +159,10 @@ internal FluentBindingRule<TAttribute> SetPostResolveHook(Func<TAttribute, Param
         /// <typeparam name="TType"></typeparam>
         /// <param name="builderInstance"></param>
         /// <returns></returns>
-        public void BindToInput<TType>(IConverter<TAttribute, TType> builderInstance)
+        public FluentBindToInputOptions BindToInput<TType>(IConverter<TAttribute, TType> builderInstance)
```

Comment:
Having a BindToInput method return an "options" class doesn't seem right

Reply:
This is fluent. Nobody will see the typenames, just the "rule.BindToInput().AddAvalidator()" etc.  I'll rename the type to whatever you want. 

---
In reply to: [151525218](https://github.com/Azure/azure-webjobs-sdk/pull/1422#discussion_r151525218) [](ancestors = 151525218)